.github/workflows/commonmark-lint.yaml

```diff
@@ -0,0 +1,25 @@
+name: "Commonmark/Markdown: Lint"
+
+# This workflow is triggered from the main CI workflow.
```

Comment:
I might be missing something, but where does it get triggered?

Reply:
Good catch